bugwarrior/command.py

```diff
@@ -67,6 +74,8 @@ def pull(dry_run, flavor, interactive, debug):
         )
     except RuntimeError as e:
         log.critical("Aborted (%s)" % e)
+    except SystemExit:
+        pass
```

Comment:
I think this defeats the purpose of `sys.exit(1)` because when you `pass` on the exception the exit code ends up being 0. Why is this necessary?